app/src/test/java/com/github/onlynotesswent/utils/NotesToFlashcardTest.kt

```diff
@@ -176,19 +180,26 @@ class NotesToFlashcardTest {
 
   @Mock private lateinit var mockContext: Context
 
-  private val savedFlashcards = mutableListOf<Flashcard>()
+  // Saved objects, use synchronizedList to avoid concurrent modification exceptions
+  private val savedFlashcards = Collections.synchronizedList(mutableListOf<Flashcard>())
```

Comment:
Well played using synchronizedList, this is really good a should solve the problem